plugins/modules/ntnx_protection_policies_v2.py

```diff
@@ -0,0 +1,764 @@
+          start_time:
+            description:
+              - Specifies the start time of the schedule.
+            type: str
+            required: false
+          sync_replication_auto_suspend_timeout_seconds:
+            description:
+              - Specifies the timeout in seconds for the sync replication to auto suspend.
+              - Auto suspend timeout if there is a connection failure between locations for synchronous replication.
+            type: int
+            required: false
+  category_ids:
+    description:
+      - Specifies the list of external identifiers of categories that must be added to the protection policy.
+    type: list
+    elements: str
+    required: false
+extends_documentation_fragment:
+  - nutanix.ncp.ntnx_credentials
+  - nutanix.ncp.ntnx_operations_v2
+author:
+  - George Ghawali (@george-ghawali)
+"""
+
+EXAMPLES = r"""
+- name: Create linear protection policy
+  nutanix.ncp.ntnx_protection_policies_v2:
+    nutanix_host: "{{ ip }}"
+    nutanix_username: "{{ username }}"
+    nutanix_password: "{{ password }}"
+    name: "linear-protection-policy-name"
+    description: "linear-protection-policy-description"
+    replication_locations:
+      - label: "ansible-label-linear-label1"
+        domain_manager_ext_id: "00000000-0000-0000-0000-000000000000"
+        is_primary: true
+      - label: "ansible-label-linear-label2"
+        domain_manager_ext_id: "11111111-1111-1111-1111-111111111111"
+        is_primary: false
+    replication_configurations:
+      - source_location_label: "ansible-label-linear-label1"
+        remote_location_label: "ansible-label-linear-label2"
+        schedule:
+          recovery_point_type: "CRASH_CONSISTENT"
+          recovery_point_objective_time_seconds: 3600
+          retention:
+            linear_retention:
+              local: 1
+              remote: 1
+          start_time: "16h:12m"
+          sync_replication_auto_suspend_timeout_seconds: 300
+      - source_location_label: "ansible-label-linear-label2"
+        remote_location_label: "ansible-label-linear-label1"
+        schedule:
+          recovery_point_type: "CRASH_CONSISTENT"
+          recovery_point_objective_time_seconds: 3600
+          retention:
+            linear_retention:
+              local: 1
+              remote: 1
+          start_time: "16h:12m"
+          sync_replication_auto_suspend_timeout_seconds: 300
+    category_ids:
+      - "22222222-2222-2222-2222-222222222222"
+    register: result
+
+- name: Create auto retention protection policy
+  nutanix.ncp.ntnx_protection_policies_v2:
+    nutanix_host: "{{ ip }}"
+    nutanix_username: "{{ username }}"
+    nutanix_password: "{{ password }}"
+    name: "auto-retention-protection-policy-name"
+    description: "auto-retention-protection-policy-description"
+    replication_locations:
+      - label: "ansible-label-auto-label1"
+        domain_manager_ext_id: "00000000-0000-0000-0000-000000000000"
```

Comment:
please mention in docs what domain_manager_ext_id is and how it can be fetched. And its significance